tools/logparser2.py

```diff
@@ -22,7 +22,11 @@ def sensor_gen(path, *selected):
                 streams.append(_stream(log, sensor))
 
         for a in _merge(streams):
-            yield a
+            if a[1] == 'camera':
+                img = log.read(os.path.basename(a[2][0]))
```

Comment:
maybe add comment how `a` for camera log looks like? I.e. what is `a[0]` for example?

Reply:
I have renamed the variables.